.travis.yml

```diff
@@ -2,5 +2,6 @@ rvm:
   - 2.3.0
 env:
   - CONCURRENCY=celluloid-io
+  - CONCURRENCY=celluloid-io WITH_GIPHY=true
```

Comment:
Thought about that as well, but since `CONCURRENCY` is a requirement and `giphy` is optional, I think the `WITH_` prefix makes it's purpose a bit more clear.